app/components/assets/assets-index/advanced-asset-columns.tsx

```diff
@@ -227,6 +227,13 @@ export function AdvancedIndexColumn({
 
     case "availableToBook":
       return <TextColumn value={item.availableToBook ? "Yes" : "No"} />;
+
+    case "upcomingReminder":
+      return (
+        <TextColumn
```

Comment:
Lets update this and make it with a popover on hover and link to asset/reminders on click.
The popover should show the name and message of the reminder. I have updated the sql query to include those.